compiler/tests/evm_contracts.rs

```diff
@@ -749,6 +749,13 @@ fn strings() {
             Some(string_token("string 5")),
         );
 
+        harness.test_function(
+            &mut executor,
+            "return_static_string",
+            vec![],
+            Some(string_token("The quick brown fox jumps over the lazy dog")),
```

Comment:
The longer string also confirms that data strings can indeed be longer than 32 bytes.